cve_bin_tool/cvedb.py

```diff
@@ -552,6 +553,53 @@ def populate_affected(self, affected_data, cursor, data_source):
         except Exception as e:
             LOGGER.info(f"Unable to insert data for {data_source} - {e}")
 
+    def metric_finder(self, cursor, cve):
+        # SQL query to retrieve the metrics_name based on the metrics_id
+        query = """
+        SELECT metrics_id FROM metrics
+        WHERE metrics_id=?
+        """
+        metric = None
+        if cve["CVSS_version"] == "unknown":
+            metric = "unknown"
+        else:
+            cursor.execute(query, [cve.get("CVSS_version")])
+            # Fetch all the results of the query and use 'map' to extract only the 'metrics_name' from the result
+            metric = list(map(lambda x: x[0], cursor.fetchall()))
+            # Since the query is expected to return a single result, extract the first item from the list and store it in 'metric'
+            metric = metric[0]
+        return metric
+
+    def populate_cve_metrics(self, severity_data, cursor):
+        insert_cve_metrics = self.INSERT_QUERIES["insert_cve_metrics"]
+
+        for cve in severity_data:
+            # Check no None values
+            if not bool(cve.get("score")):
```

Comment:
Won't cve["score"] = cve.get("score","unknown") work?

Reply:
Well, for this. I tried to keep it as similar as possible to `populate_severity`.